cpp/ql/test/library-tests/ir/ir/raw_ir.expected

```diff
@@ -11647,6 +11844,464 @@ ir.cpp:
 # 2109|     v2109_11(void)          = AliasedUse                     : ~m?
 # 2109|     v2109_12(void)          = ExitFunction                   : 
 
+# 2115| void TryCatchDestructors(bool)
```

Comment:
This is a great test! It looks like we're currently only emitting destructor calls on the non-exceptional blocks. This makes a lot of sense since it's would be a fair bit of work to ensure that we generate a destructor call on the exceptional path so I think simply not emitting them is a good choice.

Am I correct in this observation?

Reply:
We should create an issue for this, then 👍. I'll do that

Reply:
Created an issue for this now. See [this internal link](https://github.com/github/codeql-c-team/issues/2211).